README.md

```diff
@@ -4,10 +4,13 @@ Our vision is an open source software ecosystem where the time to fix a vulnerab
 
 ## Objectives and Key Results (CY 2020)
 
-The first objectives we're using to track our progress towards that vision are:
+The OpenSSF Vulnerability Disclosures Working Group seeks to help improve the overall security of the open source software ecosystem by helping mature and advocate well-managed vulnerability reporting and communication.  We plan on address this challenge through the following actions:
 
-- Create a unified format and API for vulnerability reporting (from researchers to maintainers) and drive broad adoption of it across the open source software ecosystem
-- Create a unified format, API, and process for coordinated disclosure (from maintainers to users/the world) and drive broad adoption
+-Documenting and promoting reasonable vulnerability disclosure and coordination practices within the OSS ecosystem for component maintainers and community members by providing documented standards and educational materials.
+
+- Identifying vulnerability disclosure pain points for OSS maintainer and consumers and take steps to address them
```

Comment:
In addition to identifying pain points, I'd propose adding a bit about defining and articulating ideal end-to-end workflows and automation for each persona, unencumbered by the current state of standards or technologies.  Something to the effect of:

`Identify vulnerability disclosure pain points and define ideal end-to-end workflows and automation for OSS maintainers, consumers, and security researchers.`